azavea/raster-vision
# Multiple fixes and improvements [WIP]

## Overview

Brief description of what this PR does, and why it is needed.

### Checklist


### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
* Prefer bulleted description
* Start after checking out this branch
* Include any setup required, such as rebuilding the Docker image.
* Include test case, and expected output if not captured by automated tests.

Fixes #1271
